packages/keyboard-shortcuts/src/context.js

```diff
@@ -3,4 +3,26 @@
  */
 import { createContext } from '@wordpress/element';
 
-export const context = createContext();
+const globalShortcuts = new Set();
+const globalListener = ( event ) => {
+	for ( const keyboardShortcut of globalShortcuts ) {
+		keyboardShortcut( event );
+	}
+};
+
+export const context = createContext( {
+	current: {
```

Comment:
I think we can eliminate the `.current` nested object. This is not a ref, it's an object exposed through a context.